### PR TITLE
introduce man page ots-sanitize.1

### PR DESCRIPTION
**Pull request by MicroJoe**

### CODE REVIEW

File: docs/ots-sanitize.1 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.PP
+ots-sanitize is a program which validates and/or transcodes a font file using
+the OTS library:
+.PP
+.RS
+.nf
+transcoded_font = ValidateAndTranscode(original_font);
+if (validation_error)
+  PrintErrorAndExit;
+OutputToStdout(transcoded_font);
```
Comment (khaledhosny) [inline docs/ots-sanitize.1:18]: Not sure why this pseudo code is here, without even a mention what it is, but I find it strange for man page.

**Review by khaledhosny — CHANGES_REQUESTED** [review]

**Comment by khaledhosny** [comment]
If we are providing a man page, then we should install it as well.

File: docs/ots-sanitize.1 (hunk 1 above)
Comment (MicroJoe) [reply docs/ots-sanitize.1]: This was copy-pasted from `DesignDoc.md`, removed in new patchset.

**Comment by MicroJoe** [comment]
> If we are providing a man page, then we should install it as well.

Done, installation is done via new instruction in meson.build.

**Review by khaledhosny — APPROVED** [review]

**Pull request merged**